lint.sh

```diff
@@ -77,6 +77,26 @@ check_bash(){
   done < <(find ${fargs[@]} -iname \*.sh)
 }
 
+check_naming_standards() {
+  # Limits lint to only gating unique files
+  # Excludes webhooktranslator as it has additional packaging
+  # and it's own tox
+  dirs_to_lint="pipeline_steps rpc_jobs scripts"
+  for _dir in ${dirs_to_lint}; do
+    for _file in $(find ${_dir} -type f -not -path '.*/.*'); do
+      python scripts/lint_naming_conventions.py \
+        --filename ${_file} \
+        || { echo "File naming fail: ${_file}"; rc=1; }
+    done
+  done
+
+  for _file in $(find rpc_jobs/ -type f); do
+    python scripts/lint_naming_conventions.py \
```

Comment:
I don't have a problem with this as bash is not suitable for yaml parsing. Otoh if you did do the tree walk in python it could check job names on all yaml files based on extension.